tests/integration/server/users_api_tests.py

```diff
@@ -50,3 +51,21 @@ def test_invalid_duplicate_username():
         assert response.json()["message"] == expected["message"]
         assert response.json()["code"] == expected["code"]
         delete_user_by_username(user_input["username"])
+
+
+def test_delete_user_by_next_id():
+    """Test that a user can be deleted by userid."""
```

Comment:
You refer it to userid here, it would be best to stick to next_id to keep the code more consistent.